internals/cli/masker/writer.go

```diff
@@ -85,61 +85,72 @@ type MaskedWriter struct {
 	matchers   []matcher
 	timeout    time.Duration
 
-	buf    []maskByte
-	lock   *sync.Mutex
-	output chan []maskByte
-	err    error
-	nIn    int64
-	nOut   int64
+	buf              []maskByte
+	incomingBytes    chan []byte
+	forceFlushBuffer chan struct{}
+	output           chan []maskByte
+	err              chan error
+	wg               sync.WaitGroup
 }
 
 // NewMaskedWriter returns a new MaskedWriter that masks all occurrences of sequences in masks with maskString.
 func NewMaskedWriter(w io.Writer, masks [][]byte, maskString string, timeout time.Duration) *MaskedWriter {
```

Comment:
I'd expect `maskString` to be called `mask` and `masks` to be called `sequencesToMask` or `bytesToMask`.